docs/changes.rst

```diff
@@ -1124,9 +1124,9 @@ Codename: Read all about it.
 
 Core hypothesis:
 
-* Add support back in for pypy and python 3.2
+* Add support back in for PyPy and python 3.2
 * @given functions can now be invoked with some arguments explicitly provided. If all arguments that hypothesis would have provided are passed in then no falsification is run.
-* Related to the above, this means that you can now use pytest fixtures and mark.parametrize with Hypothesis without either interfering with the other.
+* Related to the above, this means that you can now use pytest fixtures and mark.parameterise with Hypothesis without either interfering with the other.
```

Comment:
`parametrize` is a real word, apart from that the "correct" functionname doesn't exist yet.